.github/workflows/ci.yml

```diff
@@ -14,6 +14,22 @@ jobs:
   # Note that vcpkg dependencies takes the majority of the build time.
   # We cache them using GitHub Actions cache and export, making the scripts below a bit more complex.
 
+  # Format file with instruction in ~/.editorconfig file
```

Comment:
We want to enforce the consistency of code formatting, not to format — Could you please update the comment to reflect that?

Reply:
We could use this comment taken verbatim from [Consul.NET](https://github.com/G-Research/consuldotnet):
> Enforces the consistency of code formatting using `.editorconfig` and the `dotnet-format` tool.